cmd/deploy.go

```diff
@@ -26,7 +26,7 @@ import (
 
 // deployCmd represents the deploy command
 var deployCmd = &cobra.Command{
-	Use:   "deploy <stack_name> <filename>",
+	Use:   "deploy <stack_name> <filename> [extra_args]",
```

Comment:
Reading the divergence of the code and the help we can give users here... I'm wondering if it's really helpful to have a single `deploy` command, vs. separate `deploy-ethereum` and `deploy-fabric` (or `deploy ethereum`/`deploy fabric`) commands with individual args/help.

Reply:
I like the subcommand approach. So I the command would become:

```
ff deploy fabric dev my_chaincode_package.tar.gz channel_name chaincode_name
```